### ISSUE TITLE
Reproduce results gap on 3DPW / EMDB-1

### ISSUE THREAD

**Issue by anon123x** [fanegg/Human3R#11]
Hi, thanks for your great work!

I retrained the model using the provided settings for 40 epochs and evaluated the best checkpoint, but the results differ from those reported in the paper (see attached comparison for 3DPW and EMDB-1).

Could you please share the exact training configuration used for the reported table — including the number of epochs, learning rate schedule, and loss weights?

<img width="3318" height="1768" alt="Image" src="https://github.com/user-attachments/assets/1247f3fc-6ea9-415b-a52c-4e754e5be296" />
